public/app/features/explore/LogsVolumePanel.tsx

```diff
@@ -121,27 +108,18 @@ const getStyles = (theme: GrafanaTheme2) => {
       justify-content: end;
       position: absolute;
       right: 5px;
-      top: 5px;
+      top: -10px;
+      font-size: ${theme.typography.size.sm};
```

Comment:
`theme.typography.size` is deprecated, shall we use
```suggestion
      font-size: ${theme.typography.bodySmall.fontSize};
```
instead?

Reply:
Updated in d74a32bdad4bef77e184f485fc71de02c16bcc48